plugins/modules/rds_cluster.py

```diff
@@ -25,9 +25,10 @@
   # General module options
     state:
         description: Whether the snapshot should exist or not.
```

Comment:
Can you please update the description? From my understanding, `started` and `stopped` can only be sued with aurora clusters, am I wrong? Can you also update the description with something like: "C(started) and C(stopped) can only be used with Aurora DB clusters."